graylog2-server/src/main/java/org/graylog/security/shares/DefaultGranteeService.java

```diff
@@ -80,12 +93,20 @@ public DefaultGranteeService(UserService userService, GRNRegistry grnRegistry, G
                 .collect(ImmutableSet.toImmutableSet());
     }
 
+    private ImmutableSet<AvailableGrantee> getGlobalGrantees() {
+        final ImmutableSet.Builder<AvailableGrantee> builder = ImmutableSet.<AvailableGrantee>builder();
+        final Optional<UserAndTeamsConfig> config = Optional.ofNullable(clusterConfigService.get(UserAndTeamsConfig.class));
```

Comment:
same here. `getOrDefault()` makes things easier

Reply:
Yes, that is a nice idiom